crates/yas_core/src/tests/utils/swap_cases.cairo

```diff
@@ -309,116 +313,116 @@ mod SwapTestHelper {
     }
 
     fn SWAP_CASES() -> Array<SwapTestCase> {
-        array![
-            SwapTestCase {
+        array![ //16 SWAPS
+            SwapTestCase { //done
```

Comment:
We can eliminate the comments where we indicate the state of each pool, as this is documented